<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Title does not match with expectedString]]>
    </line>
    <line>
      <![CDATA[=============================================]]>
    </line>
    <line>
      <![CDATA[Page Contains ExpectedFooterText]]>
    </line>
    <line>
      <![CDATA[==============================================]]>
    </line>
    <line>
      <![CDATA[======Epic sadface: Username and password do not match any user in this service======]]>
    </line>
  </reporter-output>
  <suite started-at="2023-10-05T10:37:05 IST" name="Test_Automation_Suite" finished-at="2023-10-05T10:37:32 IST" duration-ms="27156">
    <groups>
    </groups>
    <test started-at="2023-10-05T10:37:05 IST" name="Test" finished-at="2023-10-05T10:37:32 IST" duration-ms="27156">
      <class name="com.qa.testScripts.SauceDemoTest">
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:com.qa.testScripts.SauceDemoTest@52bf72b5]" started-at="2023-10-05T10:37:06 IST" name="setUp" finished-at="2023-10-05T10:37:15 IST" duration-ms="9496" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://www.saucedemo.com/]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="sauceLoginPositiveTest(java.lang.String,java.lang.String)[pri:1, instance:com.qa.testScripts.SauceDemoTest@52bf72b5]" started-at="2023-10-05T10:37:15 IST" name="sauceLoginPositiveTest" finished-at="2023-10-05T10:37:22 IST" duration-ms="6747" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [Swage Labs] but found [Swag Labs]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Swage Labs] but found [Swag Labs]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.qa.testScripts.SauceDemoTest.sauceLoginPositiveTest(SauceDemoTest.java:74)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:136)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:658)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:219)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:923)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:192)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:32)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:467)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1623)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Title does not match with expectedString]]>
            </line>
            <line>
              <![CDATA[=============================================]]>
            </line>
            <line>
              <![CDATA[Page Contains ExpectedFooterText]]>
            </line>
            <line>
              <![CDATA[==============================================]]>
            </line>
          </reporter-output>
        </test-method> <!-- sauceLoginPositiveTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.testScripts.SauceDemoTest@52bf72b5]" started-at="2023-10-05T10:37:22 IST" name="tearDown" finished-at="2023-10-05T10:37:23 IST" duration-ms="651" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:com.qa.testScripts.SauceDemoTest@52bf72b5]" started-at="2023-10-05T10:37:23 IST" name="setUp" finished-at="2023-10-05T10:37:26 IST" duration-ms="3321" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://www.saucedemo.com/]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="sauceLoginNegativeTest(java.lang.String,java.lang.String)[pri:2, instance:com.qa.testScripts.SauceDemoTest@52bf72b5]" started-at="2023-10-05T10:37:26 IST" name="sauceLoginNegativeTest" finished-at="2023-10-05T10:37:32 IST" duration-ms="5965" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_Invaliduser]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_Invalidpwd]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[======Epic sadface: Username and password do not match any user in this service======]]>
            </line>
          </reporter-output>
        </test-method> <!-- sauceLoginNegativeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.testScripts.SauceDemoTest@52bf72b5]" started-at="2023-10-05T10:37:32 IST" name="tearDown" finished-at="2023-10-05T10:37:32 IST" duration-ms="484" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.testScripts.SauceDemoTest -->
    </test> <!-- Test -->
  </suite> <!-- Test_Automation_Suite -->
</testng-results>
